lib/factor/ConstantFactors.js

```diff
@@ -43,8 +43,11 @@ ConstantFactors.getPrimeFactors = function(number){
 ConstantFactors.getFactorPairs = function(number){
   const factors = [];
 
-  const root = Math.sqrt(number);
-  for (var divisor = 1; divisor <= root; divisor++) {
+  const bound = Math.floor(Math.sqrt(Math.abs(number)));
+  for (var divisor = -bound; divisor <= bound; divisor++) {
```

Comment:
weren't we using this factor functionality for something else? this still works there with this change?